src/components/dialogs/copy-to-script-dialog.js

```diff
@@ -57,23 +76,29 @@ const CopyToScriptDialog = ({
         >
             <DialogTitle>{title}</DialogTitle>
             <DialogContent>
-                <InputLabel htmlFor="newName">
-                    <FormattedMessage id="newNameList" />
-                </InputLabel>
-                <TextField
-                    autoFocus
-                    value={newNameValue}
-                    required={true}
-                    onChange={updateNameValue}
-                />
+                {loading ? (
+                    <CircularProgress />
+                ) : (
+                    <NameWrapper
+                        initialValue={generatedName}
```

Comment:
I would fix the initialValue behavior and restore the value={...} and onChange{...} prop